### ISSUE TITLE
vue-router v3.0.2 causes MdRouterLinkProps to fail

### ISSUE THREAD

**Issue by IanSavchenko** [vuematerial/vue-material#1977]
After updating `vue-router` to version 3.0.2 a utility  [`MdRouterLinkProps`](https://github.com/vuematerial/vue-material/blob/dev/src/core/utils/MdRouterLinkProps.js) fails. I don't invoke it directly but it is used somewhere internally. 

I found the reason: `router-link` is now registered with a name in a Pascal case, whereas here it is expected to be in kebab case. See these changes https://github.com/vuejs/vue-router/pull/1842/files.

Possible workaround: 
```js
Vue.use(VueRouter);

Vue.component('router-link', Vue.options.components.RouterLink);
Vue.component('router-view', Vue.options.components.RouterView);
```

**Comment by didimitrie** [comment]
Thanks, this helped! Hope it's going to get merged...

**Comment by guillaumemaka** [comment]
This fix is not present in version 1.0.0-beta-2 I have the same issue with <md-button> component, how to get this fix ?
